SeriesCruncher/app/src/main/java/com/balazs_csernai/seriescruncher/seriesdetails/component/SeriesDetailsComponent.java

```diff
@@ -23,7 +24,8 @@
                 SeriesDetailsModule.class,
                 ConverterModule.class,
                 ImageLoaderModule.class,
-                ColorModule.class
+                ColorModule.class,
```

Comment:
Please consider creating a common module for the image loader, the color provider, and the blur.